app/controllers/team/projects_controller.rb

```diff
@@ -23,4 +23,25 @@ def users
 
     @pager, @users = pagy_array(ByTeamProjectFetcherService.new(current_user.team).users)
   end
+
+  def by_user
+    authorize Project, :index?
+
+    user_id = params[:user_id]
+    @user = User.find(user_id)
+    @pager, @projects = pagy(Conversion::Project.assigned_to(@user)
+                                                .in_progress
+                                                .by_conversion_date, items: 10)
```

Comment:
The pager doesn't show up on the view we are using, which is not part of this work, but as a general rule we don't include `items: 10` on tabular views, it defaults to 20.

I'll make a not to add a pager to this view later. If we drop items here, we can crack on!

Reply:
Oh that's odd, I copied this `page: 10` from another view (can't remember which). I'll remove it.